test/IndexTest.cpp

```diff
@@ -287,38 +289,38 @@ TEST(IndexTest, createFromTsvTest) {
     // Lhs info
     //    ASSERT_EQ(0u, *reinterpret_cast<Id*>(buf + bytesDone));
     //    bytesDone += sizeof(Id);
-    //    ASSERT_EQ(index._posMeta.getRmd(7)._rmdBlocks->_startRhs,
+    //    ASSERT_EQ(index.POS().metaData.getRmd(7)._rmdBlocks->_startRhs,
```

Comment:
I think we should remove the commented code outright, `git` will remember it in case we ever need it again